### DISCUSSION TITLE
MHIG or just MHI2?

### DISCUSSION THREAD

**Discussion by ismisecaoimhin** [Mr-MIBonk/M.I.B._More-Incredible-Bash#333] [Q&A]
HI,
Newbie here - keen to learn!
I have a 2014 A3 Cabrio. S/w version is: MHIG_RW_AU_S1539, ,which I believe if Mib 1.
I can't reconcile the two statements below from different pages on this site:
1. "M.I.B. is running on Harman **MHIG** (MIB1 High) and MHI2/MHI2Q (MIB2 High) units only."
2. "M.I.B is made to run on the so called VAG **MIB2** Harman High units. These are used under different names on many AG cars starting **2015** until 2021+. "
Can someone confirm I can install/use MIB which does NOT appear to be a MIB2 device?

**Comment by NishilAmin1213** [comment]
Hi, I am also interested in the answer to this question as I would be running this on a 2015 Plate A3 e-tron (MHIG - MIB1).
